src/traces/treemap/attributes.js

```diff
@@ -139,29 +139,20 @@ module.exports = {
 
         colors: sunburstAttrs.marker.colors,
 
-        opacitybase: {
-            valType: 'number',
-            editType: 'style',
-            role: 'style',
-            min: 0,
-            max: 1,
-            dflt: 0.5,
-            description: [
-                'Sets the base opacity of the headers based on the depth from the entry.',
-                'This options is not available when having a `colorscale`.'
-            ].join(' ')
-        },
-
-        opacitystep: {
-            valType: 'number',
+        depthfade: {
+            valType: 'enumerated',
+            values: [true, false, 'reversed'],
             editType: 'style',
             role: 'style',
-            min: 0,
-            max: 1,
-            dflt: 0.5,
             description: [
-                'Sets the increment for opacity of the headers based on the depth from the entry.',
-                'This options is not available when having a `colorscale`.'
+                'Fades headers towards the background. Background color with opacity of 0.75',
+                'is used as the destination fading color.',
+                'When `marker.colors` are not set within the trace it is defaulted to *false*;',
+                'otherwise it is defaulted to *true*.',
```

Comment:
Hmm. This should say: This option is unavailable when a `colorscale` is present, defaults to *false* when `marker.colors` is set, but otherwise defaults to *true*.